### PR TITLE
Updating the SDK branding to 1.1.6

### PR DESCRIPTION
**Pull request by livarcocc**

### CODE REVIEW

File: build/Microsoft.DotNet.Cli.Monikers.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SdkBrandName>.NET Core SDK 1.1.5</SdkBrandName>
+    <SdkBrandName>.NET Core SDK 1.1.6</SdkBrandName>
     <SharedFrameworkBrandName>Microsoft .NET Core 1.1.5 - Runtime</SharedFrameworkBrandName>
```
Comment (livarcocc) [inline build/Microsoft.DotNet.Cli.Monikers.props]: @leecow Is the runtime changing to 1.1.6 for this?

Comment (JunTaoLuo) [reply build/Microsoft.DotNet.Cli.Monikers.props]: Runtime shouldn't change right? It was just aspnetcore and later that were rev'ed, I'll let lee confirm though.

Comment (leecow) [reply build/Microsoft.DotNet.Cli.Monikers.props]: Correct, no runtime update for this.

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora24 x64 Release Build

**Pull request merged**